apps/sessions-demo/src/config/server.ts

```diff
@@ -39,8 +39,7 @@ export const RPC = envWithDefaults("RPC", {
   testnet: "https://testnet.fogo.io",
 });
 
-export const FOGO_SESSIONS_DOMAIN =
-  process.env.FOGO_SESSIONS_DOMAIN ?? "https://sessions-example.fogo.io";
+export const FOGO_SESSIONS_DOMAIN = process.env.FOGO_SESSIONS_DOMAIN;
```

Comment:
```suggestion
export const FOGO_SESSIONS_DOMAIN = envWithDefaults("FOGO_SESSIONS_DOMAIN", {
  localnet: undefined,
  testnet: "https://sessions-example.fogo.io"
});
```

Otherwise the testnet paymaster rejects everything...